runners/flink/runner/pom.xml

```diff
@@ -101,6 +101,17 @@
 
     <dependency>
       <groupId>org.apache.beam</groupId>
+      <artifactId>runners-core</artifactId>
```

Comment:
Short answer: Flink runner uses `GroupAlsoByWindowViaWindowSetDoFn`.

Long answer: The intention for `runners/core` is to contain all the code from the SDK that is not relevant for users, but just for executing a pipeline. In particular, all the GABW implementations, the `GroupByKeyViaGroupByKeyOnly` helper, most `DoFnRunner` implementations, `SideInputReader` and implementations, and `ReduceFnRunner`. It should be a good majority of the SDK's `util` package.

I deliberately chose to move the class I did just to establish a couple dependencies so this new module wasn't entirely vacuous. Spark and Flink batch do not yet use this, but I imagine they will later.